Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-multilingual-cased and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Traceback (most recent call last):
  File "/home/sarim.hashmi/Downloads/NLP-702_assignment/bert-base-lora-finetune.py", line 185, in <module>
    main()
  File "/home/sarim.hashmi/Downloads/NLP-702_assignment/bert-base-lora-finetune.py", line 112, in main
    task_type=TaskType.SEQUENCE_CLASSIFICATION,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: type object 'TaskType' has no attribute 'SEQUENCE_CLASSIFICATION'